src/views/DidUpgradeEuro/DidUpgradeEuro.tsx

```diff
@@ -0,0 +1,137 @@
+import { browser } from 'webextension-polyfill-ts';
+
+import { Link } from 'react-router-dom';
+
+import { FormEvent, useCallback, useEffect, useState } from 'react';
+
+import ky from 'ky';
+
+import * as styles from './DidUpgradeEuro.module.css';
+
+import { Identity } from '../../utilities/identities/types';
+import { IdentitySlide } from '../../components/IdentitySlide/IdentitySlide';
+import { paths } from '../paths';
+import {
+  PasswordField,
+  usePasswordField,
+} from '../../components/PasswordField/PasswordField';
+import { LinkBack } from '../../components/LinkBack/LinkBack';
+import { getTransaction } from '../../utilities/didUpgrade/didUpgrade';
+import { endpoints, getEndpoint } from '../../utilities/endpoints/endpoints';
+import { useAsyncValue } from '../../utilities/useAsyncValue/useAsyncValue';
+
+interface Props {
+  identity: Identity;
+}
+
+// TODO: Fetch submitter address from TXD
+const submitter = '4t37z5PrEH9zz93cQ2of8F9kYMPrmWcRMBckJtNGF8keSW5W';
+
+// TODO: return TXD URL when it works to fetch submitter address
+async function getExternalURLs(): Promise<{ checkoutBaseURL: string }> {
```

Comment:
Would it be nice to have it as a mapping, like [this one](https://github.com/BTE-Trusted-Entity/sporran-extension/blob/main/src/utilities/useSubscanHost/useSubscanHost.ts)?